salt/returners/zabbix_return.py

```diff
@@ -15,20 +15,34 @@
 '''
 from __future__ import absolute_import
 import logging
+import os
 log = logging.getLogger(__name__)
 
 
+if os.path.exists('/usr/local/zabbix/bin/zabbix_sender') and os.path.exists('/usr/local/zabbix/etc/zabbix_agentd.conf'):
```

Comment:
Can you move all of this logic into the `__virtual__()` function, instead of in the global namespace here? We like to keep the global namespace as clean as possible. Then you can change the `HAS_ZABBIX` functions into simple `return True` (or False) statements.